benches/benches/block_target_gas.rs

```diff
@@ -29,6 +39,17 @@ use fuel_core_types::{
     fuel_types::AssetId,
 };
 
+#[path = "utils.rs"]
+mod utils;
+
+#[path = "block_target_gas/alu.rs"]
```

Comment:
Hmm, it already should work without `path`

Reply:
I moved the `block_target_gas` code into a new `block_target_gas/main.rs` and now it does (apparently `main.rs` is the convention for benches).